src/ate/test_programs/otlib_wrapper/src/lib.rs

```diff
@@ -329,8 +339,23 @@ pub extern "C" fn OtLibConsoleRx(
             let json_str = cap.get(1).expect("RESP_OK group").as_str();
             let crc_str = cap.get(2).expect("CRC group").as_str();
             check_console_crc(json_str, crc_str).expect("CRC check failed.");
-            msg[..json_str.len()].copy_from_slice(json_str.as_bytes());
-            *msg_size = json_str.len();
+
+            *num_frames = (json_str.len() + BUFFER_SIZE - 1) / BUFFER_SIZE;
```

Comment:
Would it be better to panic here before starting the receive?
```
if *num_frames < spi_frames.len() { /* oh noes! */ }
```

Reply:
good idea. fixed in #209.